python-sdk/docs/development/RELEASE.md

```diff
@@ -72,3 +72,29 @@ target the release branch (e.g. `release-0.7`). Failure to do these steps proper
 <!-- markdown-link-check-enable -->
 
 Once you've created your release, that's it! The rest of the steps should be handled by the CI system.
+
+# Releasing constraints for the release
```

Comment:
Did you also verify all the steps by running them once in a dummy branch?

Reply:
thanks for pointing it out. except `git tag constraints-1.4.0 && git push origin constraints-1.4.0` I have tested other command after your comment